src/swap/events.ts

```diff
@@ -5,6 +5,7 @@ export enum SwapEventName {
   SWAP_AUTOROUTER_VISUALIZATION_EXPANDED = 'Swap Autorouter Visualization Expanded',
   SWAP_DETAILS_EXPANDED = 'Swap Details Expanded',
   SWAP_ERROR = 'Swap Error',
+  SWAP_ESTIMATE_GAS_FAILURE = 'Swap Estimate Gas Call Failed',
```

Comment:
Can you renamed SWAP_ESTIMATE_GAS_FAILURE to SWAP_ESTIMATE_GAS_CALL_FAILED for consistency?